spec/game_runner_spec.rb

```diff
@@ -58,6 +57,28 @@
 
         expect(stdout.string).to include(grid_marked)
       end
-    end
+      
+      it "after the user takes a turn, the computer takes a turn, and the grid is re-rendered with an O in a random location" do
+        stdout = StringIO.new
+        player_turn = StringIO.new
+        game = GameRunner.new(stdout, player_turn)
+        expect(player_turn).to receive(:gets).and_return("C3")
+        computer_turn = srand
```

Comment:
`srand` is a great way to test the randomnesss!

I think saving it here to a variable though is confusing. If you want to make the intent clear of this line you could extract a well named method and do it at the top of the spec: `seed_random_generator_for_computer_turn`? Or something else that captures intent.

Reply:
ah, good idea! I'll do that.